assets/js/taxonomy-panel/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ *
+ * This is inspired by: https://github.com/WordPress/gutenberg/blob/1c3bc0ac022310bd0013dd572ba0368c80374806/packages/edit-post/src/components/sidebar/post-taxonomies/index.js
+ */
+import {
+	PostTaxonomies as PostTaxonomiesForm,
+	PostTaxonomiesCheck,
+} from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import TaxonomyPanel from './taxonomy-panel';
+
+function PostTaxonomies() {
```

Comment:
Is it possible to simplify this with `<HierarchicalTermSelector slug="podcasting_podcasts" />` similar to how the category selector is shown on the post-publish-check page (but without the messy checking of whether or not to display it)?  

https://github.com/WordPress/gutenberg/blob/5d5e97abd5e082050fdbb88bb1c93f9dbe10a23b/packages/editor/src/components/post-publish-panel/maybe-category-panel.js#L77-L84